Templates/Common-Backend-Scripts/utilities/dbbBuildUtils.sh

```diff
@@ -209,7 +209,17 @@ computeBuildConfiguration() {
 
 getBaselineReference() {
 
-    baselineRef=$(cat "${baselineReferenceFile}" | grep "^${mainBranchSegment}" | awk -F "=" ' { print $2 }')
+    baselienRef=""
+    
+    case $mainBranchSegment in
+        "RELEASE" | "release" | "EPIC" |  "epic")
+            baselineRef=$(cat "${baselineReferenceFile}" | grep "^${mainBranchSegment}/${secondBranchSegment}" | awk -F "=" ' { print $2 }')
+         ;;
+        "main" | "MAIN")
```

Comment:
You could translate to uppercase with `tr` instead of checking different values.